app/controllers/games_controller.rb

```diff
@@ -1,10 +1,14 @@
 class GamesController < ApplicationController
   def new
     @game = Game.new
+    @game.save
+    @game.add_white_starting_pieces!
+    @game.add_black_starting_pieces!
   end
```

Comment:
In a controller, `#new` should not be persisting anything. You can remove these lines and put them in `create`.